### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by abdellatif-laghjaj** [abdellatif-laghjaj/android-all-in-one-video-downloader#6] [Ideas]
First stable release for the all-in-one-video-downloader

<hr /><em>This discussion was created from the release <a href='https://github.com/abdellatif-laghjaj/android-all-in-one-video-downloader/releases/tag/'>v1.0.0</a>.</em>

**Comment by mohdmustaqeem7830** [comment]
unable to extract the uploader id error showing what is the solution
